CMakeLists.txt

```diff
@@ -38,15 +41,23 @@ if(CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
     install(
         FILES ${CMAKE_CURRENT_BINARY_DIR}/tmp-config.cmake
               ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindFilesystem.cmake
+        COMPONENT tmp-dev
```

Comment:
CMake suggests name such as Runtime and Development, let's stick with them

> `COMPONENT <component>`
>
> Specify an installation component name with which the install rule is associated, such as `Runtime` or `Development`. During component-specific installation only install rules associated with the given component name will be executed. During a full installation all components are installed unless marked with `EXCLUDE_FROM_ALL`. If `COMPONENT` is not provided a default component "Unspecified" is created. The default component name may be controlled with the [`CMAKE_INSTALL_DEFAULT_COMPONENT_NAME`](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_DEFAULT_COMPONENT_NAME.html#variable:CMAKE_INSTALL_DEFAULT_COMPONENT_NAME) variable.

From https://cmake.org/cmake/help/latest/command/install.html#id3